test/built-ins/decodeURI/throws-URIError.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2025 ayuan.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+info: |
+    Verify decodeURIComponent throws URIError for various invalid UTF-8 sequences
+    Reference: https://stackoverflow.com/a/1319229/172999
+esid: sec-decodeuricomponent-encodeduricomponent
+description: |
+    Invalid sequences include:
+    - Surrogate pair encoding
+    - Overlong encoding
+    - Invalid continuation bytes
+    - Incomplete sequences
+    - Out-of-range code points
+features: [uricode]
```

Comment:
No need for a `features` tag. `decodeURIComponent` is present in sufficiently old editions of ECMAScript.